components.yaml

```diff
@@ -3,25 +3,25 @@ components:
     url: https://github.com/kubevirt/bridge-marker
     commit: 6f184fa033208faf897a8fbd3aecc85811dca8bb
     branch: main
-    update-policy: tagged
+    update-policy: static
     metadata: 0.11.2
   kube-secondary-dns:
     url: https://github.com/kubevirt/kubesecondarydns
     commit: 3815007442f21ca39c7c93ead6ce4addeb9e4d29
     branch: main
-    update-policy: tagged
+    update-policy: static
     metadata: v0.0.15
   kubemacpool:
     url: https://github.com/k8snetworkplumbingwg/kubemacpool
     commit: 98d8b8233b860598ee9dd5a7ba159971e5e99be2
     branch: main
-    update-policy: tagged
+    update-policy: static
```

Comment:
dont you want to follow a branch on KMP?
either now or soon

Reply:
in next PR